misc/bazel/lfs.bzl

```diff
@@ -0,0 +1,88 @@
+def lfs_smudge(repository_ctx, srcs, extract = False, stripPrefix = None):
```

Comment:
Maybe we should call https://bazel.build/rules/lib/builtins/repository_ctx#report_progress at appropriate times?
* resolving lfs links
* local file extraction?
* downloading, unless the downloader takes care of this for us

For that, it might make sense to front-load downloading, and then extracting local files, but YMMV.